tests/ha/ntp_client.pm

```diff
@@ -10,12 +10,15 @@
 # Summary: Configure NTP client for HA tests
```

Comment:
Does this test module really have to be something HA specific?

Reply:
It was already here when I retrieved the HA tests. But yes it is not HA related, so I could easily put it in another place.
I find an already existing NTP client configuration in tests/console/yast2_ntpclient.pm, but not sure that I can use it (HA tests NTP sync on support_server).

I could create a poo# for this and work on it after, is it ok for you?

Reply:
sure

Reply:
OK, [poo#27964](https://progress.opensuse.org/issues/27964) created.